src/vms/pvm/models.ts

```diff
@@ -249,3 +249,12 @@ export interface ValidatesParams {
 export interface ValidatesResponse {
   blockchainIDs: string[];
 }
+
+export interface FeeConfigResponse {
+  weights: [number, number, number, number];
+  maxCapacity: number;
+  maxPerSecond: number;
+  targetPerSecond: number;
+  minPrice: number;
```

Comment:
Maybe we should convert these to `bigint` to easily convert to `gasPrice`, which is `bigint`. The `weights` type should match `Dimensions`